tutoraspects/templates/aspects/build/aspects/entrypoint

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+
+if [ -z "${DBT_SSH_KEY+x}"]
```

Comment:
I think you need a space before the "]". I'm seeing:

`/entrypoint: line 8: [: missing `]'`